odk1-src/aggregate-vm.rst

```diff
@@ -1,4 +1,65 @@
-ODK Aggregate Virtual Machine 
-=================================
+.. spelling::
 
-The `ODK Aggregate VM <https://gumroad.com/l/odk-aggregate-vm>`_ is a fully-configured install of Aggregate that you can run on any computer. It requires very little setup, works well without Internet connectivity, and gives you complete control over your data collection campaign.
+  readme
+  VirtualBox
+
+ODK Aggregate Virtual Machine
+=============================
+
+This document provides instructions on setting up ODK Aggregate using the `ODK Aggregate VM OVA file <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ and `VirtualBox <https://www.virtualbox.org>`_.
+
+.. admonition:: Before you get started…
+
+  - Please read this entire document before installing the virtual machine (VM). Doing so will save you a lot of time and effort.
+
+Basic Usage
```

Comment:
This headline should be something like "Installing the VM" or "Setting Up the Virtual Machine" --- it isn't actually about using it. Right?

(Also, it needs a section label.)